packages/manager/modules/navbar/src/notifications-menu/translations/Messages_fr_FR.json

```diff
@@ -1,3 +1,4 @@
 {
-  "navbar_notification_title": "Vos notifications"
+  "navbar_notification_title": "Vos notifications",
+  "navbar_notification_new": "New"
```

Comment:
I guess english is intentional here ?

Reply:
I've asked him to do it because:
- it's `NEW` in the mock-up
- the French is `Nouvelle` and it was ugly 😄

Reply:
> the French is Nouvelle and it was ugly
`Nouveau` could be enough, but no matter

Reply:
I'm not a big fan of `Nouveau` either, but I'm no the one who decides you know 😄

Reply:
@FredericEspiau Let's use `New` as it is the mock up provided content and we can improve this as it goes